src/ingest_validation_tools/table-schemas/assays/gcms-v1.yaml

```diff
@@ -0,0 +1,116 @@
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging
```

Comment:
```suggestion
      - mass_spectrometry
```

Reply:
GC & CZE mass spec are not expected to be spatial assays but may be. Therefore my recommendation is to make the assay_categories ```mass_spectrometry``` and make the spatial fields ```required: FALSE```.